test/web-platform-tests/to-upstream/dom/collections/HTMLCollection-iterator.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>HTMLCollection @@iterator Test</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+    <p id="1"></p>
+    <p id="2"></p>
+    <p id="3"></p>
+    <p id="4"></p>
+    <p id="5"></p>
+<script>
+  "use strict";
+  let paragraphs;
+  setup(() => {
+    paragraphs = document.querySelectorAll("p");
```

Comment:
No need for setup here. Just use `const paragraphs = ...` at the top of the script.